models.py

```diff
@@ -621,14 +624,44 @@ def on_model_change(self, form, model, is_created):
             msg.html = 'Thank you {}. <br/><br/><p>Please click <a href="{}/admin/reset">reset</a> and enter the registered email to reset your password.</p>'.format(model.merchant_name, app.config["SITE_URL"])
             mail.send(msg)
 
-    column_list = ['merchant_name', 'merchant_code', 'email', 'roles', 'max_settlements_per_month', 'merchant_rate', 'customer_rate', 'wallet_address']
+    column_list = ['merchant_name', 'merchant_code', 'roles', 'email', 'max_settlements_per_month', 'merchant_rate', 'customer_rate', 'wallet_address']
```

Comment:
not sure why this is swapped

Reply:
because i took it out originally when i was playing with it. and put back in the wrong place.